src/index.ts

```diff
@@ -61,6 +61,12 @@ const loadFile = (filePath: string): Record<string, unknown> => {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
       const config = require(filePath);
 
+      if (filePath.match(/config.+local/)) {
+        console.log(
+          'WARNING: Found a local config file, be cautious that this file could be published on the build phase.'
```

Comment:
This error message seems too specific to... something. I don't really know what it's referring to but we should probably just leave the message as `WARNING: Found a local config file`. We could probably also include the name of the config file it found so something like `WARNING: Found a local config file config.development.local.js`.

Reply:
Changed the error to be `WARNING: Found a local config file ${fileName} `